### PR TITLE
EC-260 - Fix Portuguese labels in the UI

### PR DESCRIPTION
**Pull request by lccpinto**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/receiver/sync/view/list/receiver-sync-message-list-view.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<th i18n="@@common-district">District</th>
 		<th i18n="@@common-operation">Operation</th>
 		<th i18n="@@common-message-id">Message Id</th>
-		<th i18n="@@commom-date-sent">Date Sent</th>
+		<th i18n="@@common-date-sent">Date Sent</th>
```
Comment (wluyima) [inline webapp/src/main/webapp/src/app/receiver/sync/view/list/receiver-sync-message-list-view.component.html:10]: Good catch!

File: webapp/src/main/webapp/src/app/receiver/receiver.component.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 			</ng-template>
 		</li>
 		<li ngbNavItem="syncMsgs">
-			<a ngbNavLink i18n="@@common-sync-msgs">Sync Items</a>
+			<a ngbNavLink i18n="@@receiver-pending-events">Sync Items</a>
```
Comment (wluyima) [inline webapp/src/main/webapp/src/app/receiver/receiver.component.html:10]: They are actually **Sync Items**, FYI the label here says **Sync Items** but in the translation file it **Pending Events**

File: webapp/src/main/webapp/src/locale/messages.pt.xlf [hunk 3]
```diff
@@ -296,6 +296,12 @@
 				<target>Eventos Pendentes</target>
 			</segment>
 		</unit>
+		<unit id="receiver-pending-events">
+			<segment>
+				<source>Pending Events</source>
```
Comment (wluyima) [inline webapp/src/main/webapp/src/locale/messages.pt.xlf:301]: They are **Sync Items** and not really events, we use events only on the sender side in some debezium events queue table

File: webapp/src/main/webapp/src/locale/messages.pt.xlf [hunk 4]
```diff
@@ -327,6 +333,12 @@
 			</segment>
 		</unit>
 		<unit id="common-sync-msgs">
+			<segment>
```
Comment (wluyima) [inline webapp/src/main/webapp/src/locale/messages.pt.xlf:336]: I'm confused here, you might as well use different english words too for this change to make sense

File: webapp/src/main/webapp/src/locale/messages.pt.xlf (hunk 3 above)
Comment (lccpinto) [reply webapp/src/main/webapp/src/locale/messages.pt.xlf]: Fixed. Thanks.

File: webapp/src/main/webapp/src/locale/messages.pt.xlf (hunk 4 above)
Comment (lccpinto) [reply webapp/src/main/webapp/src/locale/messages.pt.xlf:330]: Fixed, thanks.

**Pull request merged**